python/lancedb/embeddings/gte_mlx_model.py

```diff
@@ -0,0 +1,149 @@
+import json
+import mlx.core as mx
```

Comment:
Import mlx to try except block and raise ImportError asking users to install mlx in order to use mlx models